ajax/libs/jsnes/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "jsnes",
+  "description": "A JavaScript NES emulator",
+  "author": "Ben Firshman <ben@firshman.co.uk>",
+  "filename": "src/index.min.js",
+  "repository": {
+      "type": "git",
+      "url": "git://github.com/bfirsh/jsnes.git"
+  },
+  "license": "GPL-3.0",
+  "keywords": ["nes", "emulator", "javascript", "emulation"],
+  "npmName": "jsnes",
+  "npmFileMap": [{
```

Comment:
Let's update this to
```js
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    }
  ]
```

for
(1) The files cdnjs needs are under `dist`
(2) `**/*` is for fetching files recursively.